src/components/BookItem.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const BookItem = ({ book }) => {
+  const {
+    booktitle, author, id, category,
+  } = book;
+  return (
+    <li key={id}>
+      <p>
+        Book:
+        {booktitle}
+      </p>
+      <p>
+        Author:
+        {author}
+      </p>
+      <p>
+        Category:
+        {category}
+      </p>
+      <button type="button" bookItemID={id}>Remove</button>
+    </li>
+  );
+};
+
```

Comment:
- [ ] After you implement your changes some issues appears in the console. ♻️ 

![error-12](https://user-images.githubusercontent.com/79876959/145105591-3e01d20f-9144-4435-b775-757853d7225d.PNG)

Reply:
Done